packages/pharos-site/src/pages/getting-started.mdx

```diff
@@ -41,6 +41,13 @@ Once installed, the `node_modules/` directory should contain directories similar
 Pharos uses <PharosLink href="https://developer.mozilla.org/en-US/docs/Web/Web_Components">Web Components</PharosLink> to remain compatible with modern JavaScript frameworks, such as Vue or React.
 To accomodate patterns of each framework, the implementations vary slightly.
 
+You will first need to provide the scoped custom element registry to your application.
+This is generally achieved by including this script tag in the head of your application.
+
+```javascript
+<script src="https://cdn.jsdelivr.net/npm/@webcomponents/scoped-custom-element-registry@0.0.3/scoped-custom-element-registry.min.js"></script>
```

Comment:
The [latest version is 0.0.5](https://github.com/webcomponents/polyfills/blob/master/packages/scoped-custom-element-registry/package.json#L3). Not sure if that should be the recommendation (perhaps should be tested first).

Reply:
Use that version on a page that has at least a couple mfes registering components and ensure there aren't any console/JS errors I'd say.

Reply:
updating to version >= 0.0.4 ends up breaking some unit tests
specifically

https://github.com/ithaka/pharos/blob/docs/registry-polyfill/packages/pharos/src/components/dropdown-menu/pharos-dropdown-menu-item.test.ts#L117-L139